syntax = "proto3";
package io_channel;
option cc_enable_arenas = true;

service IoChannel {
    rpc Send(SendRequest) returns (RetCode) {}         // 发送单条数据
    rpc SendStream(stream SendRequest) returns (RetCode) {}         // 发送流数据
}

message SendRequest {
    string nodeid = 1;
    string id = 2;
    bytes data = 3;
}

message RetCode {
	int32 code = 1;
}
